detox/src/utils/retry.js

```diff
@@ -1,17 +1,19 @@
+const _ = require('lodash');
 const sleep = require('./sleep');
 
 const DEFAULT_RETRIES = 9;
 const DEFAULT_INTERVAL = 500;
 const DEFAULT_CONDITION_FN = () => true;
 const DEFAULT_BACKOFF_MODE = 'linear';
 const backoffModes = {
-  'linear': () => ({ interval, totalRetries }) => totalRetries * interval,
+  'linear': () => ({ interval, totalTries }) => totalTries * interval,
   'none': () => ({ interval }) => interval,
 };
 
-async function retry(options, func) {
-  if (typeof options === 'function') {
-    func = options;
+async function retry(optionsOrFunc, func) {
```

Comment:
rename: `options` => `optionsOrFunc` + defining `options` to have the final value (real or stub)

Reply:
perfect.